### PR TITLE
Publish Microsoft Authored templates to azure samples browser

### PR DESCRIPTION
**Pull request by v-xuto**
Publish `reliable-web-app-pattern-dotnet` template to azure samples browser.

@rajeshkamal5050 and @hemarina for notification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+- csharp
+- powershell
+- shell
+- html
+- bicep
+products:
+- azure
+- azure-application-insights
+- azure-app-service
+- azure-sql
```
[the remark this answers is not in the document]
Comment (v-xuto) [reply README.md]: Updated.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+---
+page_type: sample
+languages:
+- azdeveloper
+- csharp
+- powershell
+- shell
+- html
+- bicep
+products:
```
Comment (adrianhall) [inline README.md:10]: Add products: Redis Cache, App Configuration, Key Vault.  If the samples generally cover it, the pattern also has hub-spoke, networking, private link

File: README.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+- azure
+- azure-application-insights
+- azure-app-service
+- azure-sql-database
+- azure-front-door
+- enterprise-app-patterns
+urlFragment: reliable-web-app-pattern-dotnet
+name: Reliable web app pattern for .NET
+description: The Reliable Web App Pattern is a set of objectives to help your web application converge on the cloud. This repo contains a reference implementation of a reliable web application for .NET.
```
Comment (adrianhall) [inline README.md:19]: This repo contains a reference implementation of a reliable web application hosted in a secure hub-spoke networking configuration for .NET.

**Review by adrianhall — APPROVED** [review]

**Comment by v-xuto** [comment]
@adrianhall We have updated by your comments, please re-review.

**Review by adrianhall — APPROVED** [review]
LGTM!

**Comment by v-xuto** [comment]
@adrianhall Could you help merge this PR?

**Pull request merged**